chart/values.yaml

```diff
@@ -0,0 +1,359 @@
+      repository: ~
+      tag: ~
+  replicaCount: 1
+  podLabels: {}
+  deploymentAnnotations: {}
+  podAnnotations: {}
+  imagePullSecrets: []
+  service:
+    annotations: {}
+    labels: {}
+    type: ClusterIP
+  serviceAccount:
+    create: true
+    automount: true
+    annotations: {}
+    name: ~
+  extraContainers: []
+  extraInitContainers: []
+  extraVolumes: []
+
+back:
+  name: back
+  kyoo_migrations:
+    livenessProbe: {}
+    readinessProbe: {}
+    resources: {}
+    containerSecurityContext: {}
+    extraVolumeMounts: []
+    extraArgs: []
+    extraEnv: []
+    image:
+      repository: ~
+      tag: ~
+  kyoo_back:
+    livenessProbe: {}
+    readinessProbe: {}
+    resources: {}
+    containerSecurityContext: {}
+    extraVolumeMounts: []
+    extraArgs: []
+    extraEnv: []
+    image:
+      repository: ~
+      tag: ~
+    volumeMounts:
+      - mountPath: /metadata
+        name: back-storage
+  volumes:
+    - name: back-storage
+      persistentVolumeClaim:
+        claimName: back-storage
+  replicaCount: 1
+  podLabels: {}
+  deploymentAnnotations: {}
+  podAnnotations: {}
+  imagePullSecrets: []
+  service:
+    annotations: {}
+    labels: {}
+    type: ClusterIP
+  serviceAccount:
+    create: true
+    automount: true
+    annotations: {}
+    name: ~
+  extraContainers: []
+  extraInitContainers: []
+  extraVolumes: []
+
+front:
+  name: front
+  kyoo_front:
+    livenessProbe: {}
+    readinessProbe: {}
+    resources: {}
+    containerSecurityContext: {}
+    extraVolumeMounts: []
+    extraArgs: []
+    extraEnv: []
+    image:
+      repository: ~
+      tag: ~
+  replicaCount: 1
+  podLabels: {}
+  deploymentAnnotations: {}
+  podAnnotations: {}
+  imagePullSecrets: []
+  service:
+    annotations: {}
+    labels: {}
+    type: ClusterIP
+  serviceAccount:
+    create: true
+    automount: true
+    annotations: {}
+    name: ~
+  extraContainers: []
+  extraInitContainers: []
+  extraVolumes: []
+
+matcher:
+  name: matcher
+  kyoo_matcher:
+    livenessProbe: {}
+    readinessProbe: {}
+    resources: {}
+    containerSecurityContext: {}
+    extraVolumeMounts: []
+    # workaround until dedicated image is created
+    extraArgs: 
+      - matcher
+    extraEnv: []
+    image:
+      # workaround until dedicated image is created
+      repository: ghcr.io/zoriya/kyoo_scanner
+      tag: ~
+  replicaCount: 1
+  podLabels: {}
+  deploymentAnnotations: {}
+  podAnnotations: {}
+  imagePullSecrets: []
+  service:
+    annotations: {}
+    labels: {}
+    type: ClusterIP
+  serviceAccount:
+    create: true
+    automount: true
+    annotations: {}
+    name: ~
+  extraContainers: []
+  extraInitContainers: []
+  extraVolumes: []
+
+scanner:
+  name: scanner
+  kyoo_scanner:
+    livenessProbe: {}
+    readinessProbe: {}
+    resources: {}
+    containerSecurityContext: {}
+    extraVolumeMounts: []
+    extraArgs: []
+    extraEnv: []
+    image:
+      repository: ~
+      tag: ~
+  replicaCount: 1
```

Comment:
The scanner should never be replicated. It monitors the FS, so if two are running, it will push to rabbit events twice (like `file deleted at /vidoe/...` or `file created`).

Reply:
Added a comment stating it does not currently support multiple replicas.

Reply:
There is no plan to one day support replica for the scanner, it does not make sense to do so. I think removing the option all together would be better (with a comment explaining why it can't be replicated).